tool/travis.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+# Fast fail the script on failures.		
+set -e
+
+# Verify that the libraries are error free.
+dartanalyzer --fatal-warnings \
+  bin/analyzer.dart
```

Comment:
Done.  Soon I hope to make it "just work" for folders.  One that happens we can simplify this to ".".